tests/system/Helpers/FormHelperTest.php

```diff
@@ -810,5 +810,60 @@ public function testSetRadioDefault()
 		$this->assertEquals(' checked="checked"', set_radio('code', 'alpha', true));
 		$this->assertEquals('', set_radio('code', 'beta', false));
 	}
+	
+	
+	// ------------------------------------------------------------------------
+	public function testFormParseFormAttributesTrue()
+	{
+		$expected = 'readonly ';
+		$this->assertEquals($expected, parse_form_attributes(['readonly' => true], []));
+	}
+	
+	
+	// ------------------------------------------------------------------------
+	public function testFormParseFormAttributesFalse()
+	{
+		$expected = 'disabled ';
+		$this->assertEquals($expected, parse_form_attributes(['disabled' => false], []));
```

Comment:
Shouldn't a false setting on a value make it so that it does not appear in the attributes list?

Reply:
I thought we have more flexibility outside form helper function if we will treat true as well as false in the same way.
However I know what you are thinking so I am open to changes.

Reply:
That's fair. I can live with that, and it's still valid HTML, so I'm good. Will merge.